packages/shared/src/clients/coingecko/CoingeckoClient.ts

```diff
@@ -36,12 +36,14 @@ export class CoingeckoClient extends ClientCore {
     from: UnixTime,
     to: UnixTime,
   ): Promise<CoinMarketChartRangeData> {
+    const adjustedTo = to > UnixTime.now() ? UnixTime.now() : to
```

Comment:
isnt 
```suggestion
    const adjustedTo = Math.min(to, UnixTime.now())
```
easier to read?